builtin/credential/okta/backend.go

```diff
@@ -182,6 +182,9 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 			switch result.FactorResult {
 			case "WAITING":
 				verifyReq, err := client.NewRequest("POST", requestPath, payload)
+				if err != nil {
+					return nil, nil, nil, err
```

Comment:
See the error line just below it -- this should probably go back as the second argument, with some context around it, not the third.